E_run_front_end

```diff
@@ -0,0 +1,12 @@
+#! /bin/bash
```

Comment:
It looks like this file sneaked through, even though you have it in the .gitignore file. Please remove it from the pull request. What you may need to do is copy this file to a temporary location. Then execute:

`git rm E_run_front_end`

Then copy the temporary file back over. Then hopefully it will be on your computer but ignored by git.